pkg/controllers/proxy/network_services_controller.go

```diff
@@ -1108,13 +1168,10 @@ func parseSchedFlags(value string) schedFlags {
 		switch strings.Trim(flag, " ") {
 		case IPVS_SVC_F_SCHED1:
 			flag1 = true
-			break
```

Comment:
Not sure if those were supposed to break the for loop?

Reply:
Removing the breaks seems right to me